.gitignore

```diff
@@ -23,6 +23,7 @@
 *.sym
 *.wsp
 *~
+.vscode
```

Comment:
should be `/.vscode`  Normally I dont like to add IDE specific entries, but I see we already have one for eclipse (I also use vscode so I personally wont mind)

Reply:
Here is my wrong added, I will delete it.